src/components/filter/Filter.jsx

```diff
@@ -0,0 +1,134 @@
+import "./filter.css"
+import { Link } from "react-router-dom";
+import { useState } from "react";
+
+export default function Filter({ filterDispatch, sliderValue, productState}) {
+
+
+    const [clearSliderBtn, setClearSliderBtn] = useState(false);
+
+    const clearFilters = (e) => {
+        //dispatch CLEAR to reducer
+        filterDispatch({type: "CLEAR"})
+        //and remove clear btn from slider 
+        setClearSliderBtn(false)
+    }
+
+    const filterDispatchAndToggleClear = (e) => {
+        setClearSliderBtn(true)
+        filterDispatch({type: "SLIDER", payload: e.target.value})
+    }
+    const filterDispatchAndHideClear = () => {
+        setClearSliderBtn(false)
+        filterDispatch({type: "SLIDER", payload: 5000})
+    }
+
+    return (
+        <aside class="filter-wrapper">
+            <div class="filter-container">
+                <div class="filter">
+                    <h3>Filter</h3>
+                    <span onClick={() => clearFilters()}>Clear</span>
```

Comment:
```onClick={clearFilter}```

Reply:
Didn't realize this, thanks for pointing out